docs/terminal/basics.md

```diff
@@ -321,6 +338,10 @@ The basics of the terminal have been covered in this document. Read on to find o
 
 There's a [dedicated troubleshooting guide](/docs/supporting/troubleshoot-terminal-launch.md) for these sorts of problems.
 
+### How do I create an Admin terminal?
+
+The integrated terminal shell is running with the permissions of VS Code. If you need to run a shell command with elevated (administrator) or different permissions, use platform utilities such as `runas.exe` within a terminal.
```

Comment:
Should this point at https://code.visualstudio.com/docs/terminal/profiles#_configuring-profiles?